README.md

```diff
@@ -0,0 +1,9 @@
+# BitTitan SDK
+
+This repository contains sample scripts and tools for developers to manage and automate manipulation of BitTitan data.
+
+## Contents
+* PowerShell tools, building blocks, and full scenario scripts
+
+## Documentation
+Please visit the [BitTitan Developer Center](https://help.bittitan.com/hc/en-us/categories/115001740228-BitTitan-Developer-Center) for more inforation and resources.
```

Comment:
Future idea: this is probably the best place to add a link to some legal info (check that legal team), such as TOS, copyright, license information.

Reply:
I've taken a look at some other repositories and they seem to place all that information in a License.txt file within the root folder. 

https://github.com/BitTitan/bittitan-sdk/blob/master/LICENSE.txt

I can also place a footer in the README to reference the licensing information.